Std/Data/BitVec/Bitblast.lean

```diff
@@ -18,11 +18,8 @@ The module is named for the bit-blasting operation in an SMT solver that convert
 expressions into expressions about individual bits in each vector.
 
 ## Main results
-* `x + y : BitVec w` is equivalent to `adc x y false`.
+* `x + y : BitVec w` is `(adc x y false).2`.
 
-## Future work
-All other operations are to be PR'ed later and are already proved in
-https://github.com/mhk119/lean-smt/blob/bitvec/Smt/Data/BitVec.lean.
```

Comment:
As far as I can see, that file doesn't contain anything analogous to the contents of this file? Perhaps the comment is an orphan from an earlier period when BitVec first arrived in Std?

Reply:
Sorry, meant to link this [file](https://github.com/mhk119/lean-smt/blob/bitvec/Smt/Data/Bitwise.lean).

Reply:
Thanks.